gensim/parsing/porter.py

```diff
@@ -98,39 +132,147 @@ def _m(self):
             i += 1
 
     def _vowelinstem(self):
-        """True <=> 0,...j contains a vowel"""
+        """Check if b[i] (i = 0, ... , j) contains a vowel letter.
```

Comment:
`b[0:j + 1]` seems clearer. Also, don't forget double backticks for the class attributes.